components/brave_federated/features.cc

```diff
@@ -33,7 +33,7 @@ const int kDefaultCollectionIDLifetimeInDays = 1;
 }  // namespace
 
 const base::Feature kFederatedLearning{kFeatureName,
-                                       base::FEATURE_DISABLED_BY_DEFAULT};
+                                       base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
are you sure we are ready to enable it for everyone?

Reply:
just for testing purposes, will leave this comment open as a reminder and will change when the PR gets to a good state

Reply:
reverted it 👍

Reply:
reverted it 👍